src/test/java/seedu/address/logic/LogicManagerTest.java

```diff
@@ -24,10 +24,8 @@
 import seedu.address.logic.parser.exceptions.ParseException;
 import seedu.address.model.Model;
 import seedu.address.model.ModelManager;
-import seedu.address.model.ReadOnlyAddressBook;
 import seedu.address.model.UserPrefs;
 import seedu.address.model.person.Person;
-import seedu.address.storage.JsonAddressBookStorage;
```

Comment:
I see a few import statements with AB in the name removed, but the new ones dont seem to be imported, not sure if it matters?